LambdaEngine/Source/Game/ECS/Components/Rendering/ParticleEmitter.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Game/ECS/Components/Rendering/ParticleEmitter.h"
+
+namespace LambdaEngine {
+
+
+}
```

Comment:
Are you planning on using this file? Also, the first curly bracket is misplaced.

Reply:
Not intended